tools/NuGet/template-net60-core/DynamoVisualProgramming.ServiceCoreRuntime.nuspec

```diff
@@ -17,8 +17,6 @@
         </dependencies>
     </metadata>
     <files>
-        <!--we expect that the base path is bin\NET60_Linux\Release\-->
-        <file src="..\..\NET60_Windows\Release\**" target="lib\net6.0\windows\" />
         <file src="**" target="lib\net6.0\linux\" />
```

Comment:
we probably need to change some of the build steps too to get rid of the extra work done for delivering the windows part right ?

Reply:
also the target folder could be simplified if we only ever deliver for a single target framework/os.
Not critical though